core/modules/taxonomy/taxonomy.module

```diff
@@ -1578,7 +1585,14 @@ function taxonomy_term_title(TaxonomyTerm $term) {
 function taxonomy_field_widget_form(&$form, &$form_state, $field, $instance, $langcode, $items, $delta, $element) {
   $tags = array();
   foreach ($items as $item) {
-    $tags[$item['tid']] = isset($item['taxonomy_term']) ? $item['taxonomy_term'] : taxonomy_term_load($item['tid']);
+    // $tags[$item['tid']] = isset($item['taxonomy_term']) ? $item['taxonomy_term'] : taxonomy_term_load($item['tid']);
+    $tid = $item['tid'];
+    if ($tid == 'autocreate') {
+      $tags[] = $item;
+    }
+    else {
+      $tags[$tid] = isset($item['taxonomy_term']) ? $item['taxonomy_term'] : taxonomy_term_load($tid);
+    }
```

Comment:
Same here, seems unrelated to this PR.

Reply:
removed.
this was in the original code from Drupal, but was removed before merge. doesnt seem to be missed.